apps/etterlatte-beregning/src/main/kotlin/beregning/regler/barnepensjon/sats/BarnepensjonSats.kt

```diff
@@ -50,12 +51,12 @@ val antallSoeskenIKullet1967 =
         regelReferanse = RegelReferanse(id = "BP-BEREGNING-1967-ANTALL-SOESKEN"),
     ) benytter soeskenIKullet1967 med { soesken -> soesken.size }
 
-val avdodeForeldre2024: Regel<BarnepensjonGrunnlag, List<Folkeregisteridentifikator>> =
+val avdodeForeldre2024: Regel<BarnepensjonGrunnlag, List<String?>> =
```

Comment:
Denne opplysningen trenger vi på beregningsperioden for å eksponere dette videre til brev der vi trenger opplysningen for ukjent forelder feks

Reply:
@sebassonav dekker commit b2268c953d0167238e078f1003dc0e44adf2f7e3 det?

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4865/files

Reply:
Tror ikke det om jeg husker rett så trenger jeg denne opplysningen https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4865/files#diff-f3964489b3735a65ad9fa16443abe3096878158b93c445d6ca260d96ed647207L86

her 
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4865/files#diff-932840c95e26b87be4e26c809db66fff9b060d9b73e6acb89c08859cff9ed78bR123
Dekke den commiten det? For det ser jeg ikke helt